operator/v1/types.go

```diff
@@ -52,6 +52,33 @@ type ClusterManagerSpec struct {
 	// +optional
 	// +kubebuilder:default={mode: Default}
 	DeployOption DeployOption `json:"deployOption,omitempty"`
+
+	// Detached include configurations we needs in the detached mode.
+	// +optional
+	Detached Detached `json:"detached"`
+}
+
+// Detached represents customized configurations we need to set in the detached mode.
+type Detached struct {
+	// RegistrationAPIServiceTargetPort is used in rendering the file "manifests/cluster-manager/hub/cluster-manager-registration-webhook-apiservice.yaml".
+	//
+	// +optional
+	RegistrationWebhookAPIService APIService `json:"registrationWebhookAPIService"`
+
+	// WorkAPIWebhookAPISerivce is used in redering the file "manifests/cluster-manager/hub/cluster-manager-work-webhook-apiservice.yaml".
+	//
+	// +optional
+	WorkWebhookAPIService APIService `json:"workWebhookAPIService"`
+}
+
+// APIService represents customized configurations we need to set for APIService.
+type APIService struct {
+	// ServicePort represents the `Spec.Service.Port` in APIService.
```

Comment:
ServicePort is the port of the service that serving the apiservice

Reply:
Updated.